### PR TITLE
Add install script for those not using devcontainer

### PR DESCRIPTION
**Pull request by Porges**
This installs the tools to `$clone/hack/tools` instead. There is a `dev.sh` script to run to get into a shell with a `PATH` pointing to the right place.

### CODE REVIEW

File: .devcontainer/Dockerfile [hunk 1]
```diff
@@ -14,44 +14,21 @@ RUN apt-get update \
...
 # note that the base image includes oh-my-bash, we are enabling plugins here
 RUN sed -i '/^plugins=/a kubectl\ngolang' "/home/vscode/.bashrc"
 RUN sed -i '/^completions=/a kubectl\ngo\ntask' "/home/vscode/.bashrc"
-# make kubectl completions work with 'k' alias
+
+# Make kubectl completions work with 'k' alias
 RUN echo 'complete -F __start_kubectl k' >> "/home/vscode/.bashrc"
 
+# Setup go-task completions
+RUN curl -sL "https://raw.githubusercontent.com/go-task/task/v3.0.0/completion/bash/task.bash" > "$HOME/.oh-my-bash/completions/task.completion.sh"
```
Comment (Porges) [inline .devcontainer/Dockerfile:32]: Revert to home/vscode

**Review by Porges — COMMENTED** [review]
Needs a fix before merge

File: .devcontainer/Dockerfile (hunk 1 above)
Comment (Porges) [reply .devcontainer/Dockerfile:32]:
```suggestion
RUN curl -sL "https://raw.githubusercontent.com/go-task/task/v3.0.0/completion/bash/task.bash" > "/home/vscode/.oh-my-bash/completions/task.completion.sh"
```

**Review by babbageclunk — APPROVED** [review]
Looks awesome!

**Pull request merged**